VirtoCommerce.CatalogModule.Web/Scripts/resources/categories.js

```diff
@@ -1,7 +1,8 @@
 ﻿angular.module('virtoCommerce.catalogModule')
 .factory('virtoCommerce.catalogModule.categories', ['$resource', function ($resource) {
 
-	return $resource('api/catalog/categories/:id', { categoryId: '@Id' }, {
+    return $resource('api/catalog/categories/:id', { categoryId: '@Id' }, {
```

Comment:
This method declaration is redundant. 
Follow construction will work without this definition `categories.get({ id: blade.currentEntityId }`

Reply:
remove this get